src/Tutor.Core/ContentModel/LearningObjects/ArrangeTasks/ArrangeTaskContainerEvaluation.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace Tutor.Core.ContentModel.LearningObjects.ArrangeTasks
 {
-    public class ArrangeTaskContainerEvaluation
+    public class ArrangeTaskContainerEvaluation : AssessmentEvent
```

Comment:
Take care - each Submission has a paired Evaluation. AnswerEvaluation, ArrangeTaskContainerEvaluation, and ChallengeEvaluation.

Here you are mixing and matching different things with the AssessmentEvent. I think we should remove the assessment event for now and think about implementing it after refactoring the challenges and better understanding the Submission - Evaluation flow and how it connects to the KC.